src/components/AudioExerciseMessage.js

```diff
@@ -0,0 +1,80 @@
+import Modal from "@mui/material/Modal";
+import * as s from "./ExtensionMessage.sc";
+import Feature from "../features/Feature";
+import LocalStorage from "../assorted/LocalStorage";
+import strings from "../i18n/definitions";
+import * as t from "../exercises/exerciseTypes/Exercise.sc";
+
+export default function AudioExerciseMessage({
+  open,
+  displayedAudioExperimentPopup,
+  setDisplayedAudioExperimentPopup,
+  setAudioExerciseMessageOpen,
```

Comment:
what is the meaning of the highlighting colors? why are some props a very pale color?

<img width="565" alt="image" src="https://user-images.githubusercontent.com/464519/166825026-94b3b7af-7e9d-4923-8c83-1a676ed24240.png">

Reply:
-	They’re not being used, removed them from the code